osu.Game/Graphics/Containers/ScalingContainer.cs

```diff
@@ -101,8 +105,15 @@ private void load(OsuConfigManager osuConfig)
 
             private void scaleChanged(ValueChangedEvent<float> args)
             {
-                this.ScaleTo(new Vector2(args.NewValue), 500, Easing.Out);
-                this.ResizeTo(new Vector2(1 / args.NewValue), 500, Easing.Out);
+                this.TransformBindableTo(currentScale, args.NewValue, duration, Easing.OutQuart);
```

Comment:
Not a big deal, but you could use `this.TransformTo(nameof(currentScale))`.

Reply:
is that more preferred for some reason?

Reply:
oh, to not make it a bindable? i didn't even consider that.

Reply:
Doesn't make much of a difference at the end of the day, just avoids a bindable alloc.